lib/yard/handlers/common/method_handler.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+# Shared functionality between Ruby and C method handlers.
```

Comment:
It looks like this will add a docstring to the `YARD::Handlers` module, not MethodHandler.

Reply:
You will also want at least one newline after the frozen directive